runtime/common/src/apis.rs

```diff
@@ -418,10 +418,6 @@ macro_rules! impl_runtime_apis_plus_common {
 
 			impl session_keys_primitives::VrfApi<Block> for Runtime {
 				fn get_last_vrf_output() -> Option<<Block as BlockT>::Hash> {
-					// TODO: remove in future runtime upgrade along with storage item
```

Comment:
This will affect also the other runtimes where randomness has  not being initialized?

Reply:
Yes, it's in the PR description that this should not be merged until all runtimes have already upgraded to include randomness (moonriver, moonbeam have not yet).